docs/_articles/en/v3.x/salesforce-graph-engine/try-it-yourself.md

```diff
@@ -3,4 +3,112 @@ title: 'Try it yourself'
 lang: en
 ---
 
-## TDB from quip document
\ No newline at end of file
+## SFGE Example Executions
+
+### Intro to SFGE
+SFGE works a bit differently from other analyzers with which you might be familiar. Most static analyzers can identify
```

Comment:
How do you like this?

"Most static analyzers can identify problems that occur in a single location. For example, identifying dead code, unnecessary variables, poorly-named variables, and the like. SFGE works a bit differently from other analyzers with which you might be familiar. "